src/openfermion/resource_estimates/pbc/thc/factorizations/thc_jax.py

```diff
@@ -63,23 +63,23 @@ def load_thc_factors(chkfile_name: str) -> KPointTHC:
     xi = None
     with h5py.File(chkfile_name, "r") as fh5:
         chi = fh5["chi"][:]
-        G_mapping = fh5["G_mapping"][:]
-        num_kpts = G_mapping.shape[0]
+        g_mapping = fh5["g_mapping"][:]
```

Comment:
how was this ever working?

Reply:
I think this was some merge issue where I lowercased everything which was previously uppercase.

Reply:
so the lowercase hdf5 string is correct? it would be nice if this sort of thing was covered by a test